packages/wrangler/src/pages/upload.tsx

```diff
@@ -368,6 +368,16 @@ export const upload = async (
 	);
 };
 
+// Decode and check that the current JWT has not expired
+function isJwtExpired(token: string): boolean {
+	const decodedJwt = JSON.parse(
```

Comment:
Random `JSON.parse` errors can be cryptic depending on where in the stack they occur. Can we catch and re-throw a more specific error? That or use this package https://www.npmjs.com/package/jwt-decode